packages/core/blueprints/navi-core/index.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
         { name: 'ember-radio-button', target: '1.2.1' },
         { name: 'ember-sortable', target: '1.10.0' },
         { name: 'ember-cli-string-helpers', target: '1.8.1' },
-        { name: 'ember-tether', target: '0.4.1' },
+        { name: 'ember-tether', target: '1.0.0' },
```

Comment:
Can we add this to dependencies instead of the blueprint? We have a ticket to get rid of the blueprints anyways.

Reply:
I'm just going to add them to both. It can get removed from the blueprint when we fully move away from blueprints